<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Localcredentialdialog</class>
 <widget class="QDialog" name="Localcredentialdialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>337</width>
    <height>261</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>9</y>
     <width>321</width>
     <height>52</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>13</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="LB_Logo">
      <property name="minimumSize">
       <size>
        <width>140</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>140</width>
        <height>50</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>13</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>69</y>
     <width>327</width>
     <height>189</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QFrame" name="frame">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLabel" name="LB_CitxServer">
             <property name="text">
              <string>Server :</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="LB_CitxUsername">
             <property name="text">
              <string>Username :</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="LB_CitxPassword">
             <property name="text">
              <string>Password :</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="LB_CitxDomain">
             <property name="text">
              <string>Domain :</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLineEdit" name="LE_CitxServer">
             <property name="minimumSize">
              <size>
               <width>211</width>
               <height>27</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>213</width>
               <height>27</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="LE_CitxUsername">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>27</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>213</width>
               <height>27</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="Leforpassword" name="LE_CitxPassword" native="true">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>27</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>213</width>
               <height>27</height>
              </size>
             </property>
             <property name="focusPolicy">
              <enum>Qt::StrongFocus</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="WID_Domain" native="true">
             <property name="minimumSize">
              <size>
               <width>213</width>
               <height>27</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>213</width>
               <height>16777215</height>
              </size>
             </property>
             <widget class="QComboBox" name="CB_CitxDomain">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>213</width>
                <height>27</height>
               </rect>
              </property>
              <property name="minimumSize">
               <size>
                <width>213</width>
                <height>27</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>213</width>
                <height>27</height>
               </size>
              </property>
             </widget>
             <widget class="QLineEdit" name="LE_CitxDomain">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>200</width>
                <height>27</height>
               </rect>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>27</height>
               </size>
              </property>
             </widget>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>13</width>
          <height>26</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="PB_KbdLayout">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>27</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>27</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="PB_CitxOk">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>27</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>120</width>
            <height>27</height>
           </size>
          </property>
          <property name="text">
           <string>OK</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="PB_CitxCancle">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>27</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>120</width>
            <height>27</height>
           </size>
          </property>
          <property name="text">
           <string>Cancel</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Leforpassword</class>
   <extends>QWidget</extends>
   <header location="global">../../../LCore/leforpassword.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
